content/security/committers.md

```diff
@@ -1,110 +1,118 @@
+4. The project team sends an e-mail to the original reporter to acknowledge the report, with a copy to `security@project.apache.org` if it exists, or to
 `security@apache.org`.
 
-1. The project team investigates the report and either rejects or accepts
+5. The project team investigates the report and either rejects or accepts
 it.
 
-1. If the project team rejects the report, the team writes to the reporter to
+6. If the project team **rejects** the report, the team writes to the reporter to
 explain why, with a copy to `security@project.apache.org` if it exists, or to
 `security@apache.org`.
 
-1. If the project team accepts the report, the team writes to the reporter to let them
+7. If the project team **accepts** the report, the team writes to the reporter to let them
 know that they have accepted the report and that they are working on a fix.
 
-1. The project team requests a CVE (<a href="https://cve.mitre.org/" target="_blank">Common Vulnerabilites and Exposures</a>) name from the internal portal, `https://cveprocess.apache.org`.  Alternatively by
+8. The project team requests a CVE (<a href="https://cve.mitre.org/" target="_blank">Common Vulnerabilites and Exposures</a>) ID from the internal portal, `https://cveprocess.apache.org`; or by
 sending an e-mail with the subject "CVE request for..." to `security@apache.org`, providing a
 short (one-line) description of the vulnerability. `security@apache.org` can
-help determine if a report requires multiple CVEs or if multiple reports
-should be merged under a single CVE.
+help determine if a report requires multiple CVE IDs or if multiple reports
+should be merged under a single ID.
 
-1. The ASF security team will allocate a CVE and send a link to the
-internal portal where the project team can enter details of the
+9. The ASF security team allocates a CVE ID and sends to the project team a link to the
+internal portal where it can enter details of the
 vulnerability.
 
-1. The project team agrees on a fix on their private list.
+### Resolve
+
+10. The project team agrees on a fix on their private list.
 
-1. The project team writes up the details of the vulnerability and the fix on the
+11. The project team documents the details of the vulnerability and the fix on the
 internal portal. The portal generates draft announcement texts.  For
 an example of an announcement see [Tomcat's announcement of
 CVE-2008-2370](http://markmail.org/message/w7mdjdxeqius7d6l). The
 level of detail to include in the report is a matter of
 judgement. Generally, reports should contain enough information to
-enable people to assess the risk associated with the vulnerability for
-their own system, and no more. Announcments do not normally include steps to reproduce the vulnerability.
+enable people to assess the risk the vulnerability poses for
+their own system, and no more. Announcments do **not** normally include steps to reproduce the vulnerability.
```

Comment:
I'm not sure if this is a bold 'not'. Is this a strong instruction from the Security team or a guidance?

Reply:
updated in the patch